src/bindings/c/include/openvino/c/ov_node.h

```diff
@@ -17,12 +17,21 @@
 typedef struct ov_output_const_node ov_output_const_node_t;
 typedef struct ov_output_node ov_output_node_t;
 
+/**
+ * @struct ov_output_const_node_list_t
+ * @brief Represents an array of const ov_output_nodes.
+ */
+typedef struct {
+    ov_output_const_node_t* output_ports;
+    size_t size;
+} ov_output_const_node_list_t;
```

Comment:
Also this structure doesn't look like a list, it is just a wrapper under raw array.

Reply:
The item in this list can be accessed in C code, but only accessed in C++ scope, so we have provide helper function  if C code want to access its items.

Reply:
If we want get a C type list, maybe we can define it as below:
```
typedef struct {
    ov_output_const_node_t** ports;  // ports[i] is `ov_output_const_node_t*` data, we can use [] to access it.
    size_t size;
} ov_output_const_node_list_t;
```

Reply:
If I remember right we decided to remove list of ports

Reply:
> If I remember right we decided to remove list of ports

Yes. It has been removed in the last two commits. You can update "Files Changed".
https://github.com/openvinotoolkit/openvino/pull/12617/commits/e46a3512587d2afc5d9043ecc50ad4faa18d5e2a#diff-da88f4d18a59a15a01f64ef30541ecddc74b13c2ed9b4bebae29d53d229fcdc4